### PR TITLE
Contacts page

### PR DESCRIPTION
**Pull request by Ahmed101Mohammed**
# Changes that I do in this branch:
- Create a "style-library" folder: This folder contains the style of common components used on every page of the portfolio-website design in Figma.
- Create a "contacts" page, with the full style of the page.
- I didn't modify the "index.html" file, and "style.css", because I suggest that you may want these files to be changed for educational reasons.

### CODE REVIEW

**Comment by Ahmed101Mohammed** [comment]
Done creating contacts page, in the contacts-page branche

File: contacts.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <!-- Global and Theme styling -->
+        <link rel="stylesheet" href="/global.css">
+        <link rel="stylesheet" href="/theme.css">
+        <!-- css liberaries -->
```
Comment (hayk-simonyan) [inline contacts.html]: a small typo here "liberaries" -> libraries

File: contacts.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <!-- Global and Theme styling -->
+        <link rel="stylesheet" href="/global.css">
```
Comment (hayk-simonyan) [inline contacts.html]: no need to add the forward slash if the file is in root directory, we can use href="global.css"

File: contacts.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
+.main-container
+{
+    width: 1024px;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    gap: 25px;
+}
```
Comment (hayk-simonyan) [inline contacts.css]: To keep the code consistent with the rest of the project, I recommend using `.main-container {` format of styling (not { on the next line). You can automatically apply this by using "Find and replace" in vscode or by using an online CSS formatter

**Review by hayk-simonyan — CHANGES_REQUESTED** [review]
Great work on creating the Contacts page and splitting the CSS into separate files.

I have a couple of minor suggestions to improve the code further. Once you've addressed these small suggestions, I'd be happy to merge your changes into the main branch.

File: contacts.css (hunk 3 above)
Comment (Ahmed101Mohammed) [reply contacts.css]: Thanks so much for all of your comments. I will fix all of errors that you mention for and  repush the request♡

**Comment by Ahmed101Mohammed** [comment]
# Solving the three issues is done.
## What I change:
1. Remove all unneeded forward slash, for any path contacts.html file. (Note: There is a problem in one of my commits when I say index.html file, it's a wrong commit from me.).
2. Solve a wrong written English word for "libraries", in a comment in the contacts.html file.
3. Modify the format of CSS code in the conatcts.css file, by making the open curly bracket of each selector in the same line with its selector with one space distance, to follow the global code style of the project.

**Comment by hayk-simonyan** [comment]
> # Solving the three issues is done.
> ## What I change:
> 1. Remove all unneeded forward slash, for any path contacts.html file. (Note: There is a problem in one of my commits when I say index.html file, it's a wrong commit from me.).
> 2. Solve a wrong written English word for "libraries", in a comment in the contacts.html file.
> 3. Modify the format of CSS code in the conatcts.css file, by making the open curly bracket of each selector in the same line with its selector with one space distance, to follow the global code style of the project.

Thanks for fixing all of the issues. Look great now 👍

**Review by hayk-simonyan — APPROVED** [review]

**Pull request merged**